api/src/test/java/org/pmiops/workbench/api/CohortsControllerTest.java

```diff
@@ -140,16 +146,33 @@ public void setUp() throws Exception {
 
     CLOCK.setInstant(NOW);
     WorkspacesController workspacesController = new WorkspacesController(workspaceService,
-        cdrVersionDao, userDao, userProvider, fireCloudService, cloudStorageService, CLOCK,
+        cdrVersionDao, cohortDao, userDao, userProvider, fireCloudService, cloudStorageService, CLOCK,
         userService);
     stubGetWorkspace(WORKSPACE_NAMESPACE, WORKSPACE_NAME, "bob@gmail.com",
         WorkspaceAccessLevel.OWNER);
+    JSONObject demoCohort = new JSONObject();
+    demoCohort.put("name", "demo");
+    demoCohort.put("description", "demo");
+    demoCohort.put("type", "demo");
+    demoCohort.put("criteria", createDemoCriteria());
+    List<JSONObject> demoCohorts = new ArrayList<>();
+    demoCohorts.add(demoCohort);
```

Comment:
these two lines can be condensed
`List<JSONObject> demoCohorts = Collections.singletonList(demoCohort);`